kubernetes/kubernetes.go

```diff
@@ -30,6 +22,16 @@ import (
 	"strings"
 	"time"
 
+	"github.com/cloudability/metrics-agent/client"
```

Comment:
Looks like our imports are batching together weird.  Should we make it consistent so that they're listed together and alphabetical?

Reply:
Yeah VSCode is awesome in this - I have a go format tool installed that utilizes `goimports`. It does all the work that `gofmt` does, like making sure your code follows the Go standard, as well as removing unused imports and  organizing imports alphabetically with standard libs grouped at the top. (If you put newlines in-between imports then it'll organize each chunk alphabetically.) 

Do y'all not use that tool in GoLand? I can go around to each file and do clean up to make them all consistent if that'd be helpful. Shouldn't take too long.

Reply:
I have that, but I was thinking about the newline between groups - is that a standard we want to keep?

Reply:
Oh I see. I misunderstood your question. I have no idea why there's a newline. Happy to remove.

Reply:
I kinda like the newline between groups, shows which imports are custom vs go standard go libs. But up to y'all!

Reply:
I agree. I like the newline between the standard libs and third party packages. I think Becky means the additional newline below separating third party packages iiuc.

Reply:
I was just wondering about newlines in general.  But at the very least, we probably should only have at least one for between standard and custom packages.

Reply:
Sounds good! I'll remove and push up those changes.